packages/sources/mock-ea/src/endpoint/price.ts

```diff
@@ -0,0 +1,45 @@
+import { Validator } from '@chainlink/ea-bootstrap'
+import { ExecuteWithConfig, InputParameters } from '@chainlink/types'
+import { ExtendedConfig } from '../config'
+
+export const supportedEndpoints = ['price']
+
+export interface ResponseSchema {
+  data: {
+    price: number
+  }
+}
+
+export const inputParameters: InputParameters = {}
+
+interface ResponseInfo {
+  result: number
+  lastUpdated?: number
+}
+
+const responseInfo: ResponseInfo = {
+  result: 1000,
+}
+
+export const execute: ExecuteWithConfig<ExtendedConfig> = async (request, _, config) => {
+  const validator = new Validator(request, inputParameters)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const lastUpdated = responseInfo.lastUpdated
+  const now = Date.now()
+  if (!lastUpdated || now - lastUpdated >= config.updateIntervalInMS) {
+    responseInfo.result =
+      now % 2 === 0
```

Comment:
This won't guarantee that the result will change in the last x interval, but that it will change 50% of the time we hit the update interval.

Reply:
Could consider doing a percentage change instead?

Reply:
hmm I'm not sure I follow.  Won't the value go up or down?

Reply:
`now % 2 === 0` will only be true half the time, right? So when we hit `updateIntervalInMS`, we have a 50% chance of the result updating or not.

Reply:
right half the time we increment the result and the other half we decrement the result